db/data_migration/20150317165803_republish_case_studies_to_render_on_government_frontend.rb

```diff
@@ -0,0 +1 @@
+DataHygiene::PublishingApiRepublisher.new(CaseStudy.where(:state => Edition::PUBLICLY_VISIBLE_STATES)).perform
```

Comment:
i had suggested that we should republish unpublishings related to case studies because their `rendering_app` should now be `government-frontend`. could you confirm whether that's necessary and i'll update the PR on his behalf.